calcom/cal.com
# fix: unpublished org/team avatar and i18n

## What does this PR do?

When an unpublished org or (sub)team gets opened using their URL, the avatar was not shown correctly. Also following crowdin directives to separate i18n strings into their own for orgs/teams.

<kbd><img width="1504" alt="image" src="https://github.com/calcom/cal.com/assets/467258/239c56c5-4044-4b0b-9032-fea6e02139f2"></kbd>

## Type of change


## How should this be tested?

Create an regular team, an org or org subteam and do not publish them (having env var `IS_TEAM_BILLING_ENABLED` set to true), then see the unpublished screen come up showing the correct avatar of the team/org previously set.
